recipes/postgresadapter/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "PostgresAdapter" %}
+{% set version = "1.0.0" %}
+{% set sha256 = "e87597762d6d2b7bb79f0a84afc84f483f2569decf63e75d97c28009d1be9f4f" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name|lower }}-{{ version }}.tar.gz
+  url: https://github.com/ContinuumIO/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [py>=36]
+  script: python setup.py install
+
+requirements:
+  build:
+    - python
+    - numpy 1.11*
+    - cython
+    - postgresql
+    - psycopg2
+    - toolchain
+
+  run:
+    - python 
+    - numpy
+    - pandas
+    - six
+    - ordereddict  # [py26]
```

Comment:
We do not build for py26 anymore afaik.

Reply:
https://github.com/conda-forge/staged-recipes/pull/5141/commits/2e329a78315b903997be115d546f39c43ccbcd58